### ISSUE TITLE
bug(oem/fv/ios): insertText crash - forced unwrapping of optional `nil`

### ISSUE THREAD

**Issue by jahorton** [keymanapp/keyman#5300]
**Describe the bug**

![Screen Shot 2021-06-17 at 8 53 11 AM](https://user-images.githubusercontent.com/25213402/122318587-78222d00-cf49-11eb-89e3-0592c058344e.png)

(The most frequent issue)

Interestingly... I'm not seeing equivalent crash reports from the Keyman app.  Just from the FV app / SWKeyboard.

Digging into the files in that report's package...

```
Thread 0 name:
Thread 0 Crashed:
0   KeymanEngine                  	0x000000010137a54c Swift runtime failure: Unexpectedly found nil while unwrapping an Optional value + 0 (InputViewController.swift:374)
```

which corresponds to the final line of this block:

https://github.com/keymanapp/keyman/blob/648e336be7f00c5f281a6d4c5457228e78bd5c23/ios/engine/KMEI/KeymanEngine/Classes/InputViewController.swift#L369-L374

My reading of this strongly suggests a context desynchronization issue has arisen somehow - that KMW wants to delete text that the iOS side doesn't believe exists.  

Possibilities:
- When iOS thinks the current context is an empty string, [`.last` holds `nil`](https://developer.apple.com/documentation/swift/string/utf16view/1688980-last).
- I believe that I have seen effectively empty context before at paragraph breaks.  Though... the event count is rather high for that to be the case, especially since we aren't getting the same crashes from the Keyman app.

Again... interestingly, this is _only_ showing up in the FV product.

**To Reproduce**
_Currently unknown_.

No idea which keyboard(s) or what input is giving rise to this behavior.

**Keyman for iPhone/iPad/Android:**
 - Keyman version 14.0.274, 14.0.275
 - In system-keyboard mode

**Comment by mcdurdin** [comment]
> * hough... the event count is rather high for that to be the case,

Do you have a Sentry reference?

**Comment by jahorton** [comment]
I'll have to see if I can find a matching Sentry error.  I drew the "event count" for the quoted comment from within the first screenshot.

**Comment by jahorton** [comment]
Honestly... the crash logs obtained from Xcode are more informative here.

**Comment by sgschantz** [comment]
Fixed by PR #6371
